modules/tooltip/react/README.md

```diff
@@ -81,25 +41,37 @@ class TooltipExample extends React.Component<{}, TooltipExampleState> {
 
 ### Required
 
-> None
+#### title: string | React.ReactNode
+
+> If present, this will switch the Tooltip into a simpler to use mode where the Tooltip component
+> can wrap a target and `title` is the contents of the tooltip. This should be a string in most
+> cases. HTML is supported, but only text is understood by assistive technology. This is true for
+> both `label` and `describe` modes.
+
+#### children: React.ReactNode
+
+> The contents of the target for the Tooltip.
```

Comment:
It's not really the contents. If `children` is a button and I say "the button's contents", I think of the text, not the outer element

```suggestion
> The target (anchor element) for the Tooltip.
```

Reply:
That is more clear